src/renderer/catalog/on-run.injectable.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable, lifecycleEnum } from "@ogre-tools/injectable";
+import catalogEntityRegistryInjectable from "./entity-registry.injectable";
+
+const onRunInjectable = getInjectable({
+  instantiate: (di) => di.inject(catalogEntityRegistryInjectable).onRun,
```

Comment:
Hmm. I'm thinking if roughly this kind of delegation could be used elsewhere as well to cut wings from more `reactions` and `autoruns`. Mental note.